pkg/volume/csi/csi_mounter.go

```diff
@@ -227,7 +240,8 @@ func (c *csiMountMgr) SetUpAt(dir string, mounterArgs volume.MounterArgs) error
 	// Inject pod information into volume_attributes
 	podAttrs, err := c.podAttributes()
 	if err != nil {
-		return errors.New(log("mounter.SetUpAt failed to assemble volume attributes: %v", err))
+		opExitStatus = volumetypes.OperationStateNoChange
```

Comment:
target_dir will not be cleaned up here?

Reply:
maybe move MkdirAll after getting these attributes?

Reply:
same as above